### DISCUSSION TITLE
Question of modeling of ageing lithium-ion batteries

### DISCUSSION THREAD

**Discussion by YouHyin** [pybamm-team/PyBaMM#2816] [Q&A]
Hello, I want to simulate cycling aging of lithium-ion batteries. So I select SEI growth and lithium plating(irreversible) as cell ageing mechanisms. I really want to apply this model from  **Yang, Xiao-Guang, et al. "Modeling of lithium plating induced aging of lithium-ion batteries: Transition from linear to nonlinear aging." Journal of Power Sources 360 (2017): 28-40**. To do that, do I write the code like this? Can I consider the term in which sei growth and lithium plating interact when writing code like this?

>`options={
"SEI": "ec reaction limited",
"lithium plating": "irreversible",}
spm = pybamm.lithium_ion.SPMe(options)`

Also, when applying Yang (2017)'s model, can I apply Okane 2022's parameter to get the parameter of lithium plating? (https://github.com/pybamm-team/PyBaMM/blob/5465052c7cf2731c1bd77871b50ba67347fff19a/pybamm/input/parameters/lithium_ion/OKane2022.py)

**I'd appreciate it if you could help me.**

**Comment by valentinsulzer** [comment]
Yang 2017 purposefully keep some of their parameters secret so it's difficult to reproduce. You have to look in the paper for which ones they do give and then make educated guesses for the others
